### PR TITLE
Stories/s3 export import no key

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/grda_warehouse/recurring_hmis_export.rb [hunk 1]
```diff
@@ -66,21 +66,45 @@ def store(report)
...
+              send "#{zip_password}\r"
+            }
+          }
+          expect {
+            -nocase -re "Verify password:.*" {
+              send "#{zip_password}\r"
+            }
+          }
+          send_user "\n>> Password sent\n"
+          interact
```
Comment (awisema) [inline app/models/grda_warehouse/recurring_hmis_export.rb:97]: What do these lines do?

File: app/models/grda_warehouse/recurring_hmis_export.rb [hunk 2]
```diff
@@ -115,14 +139,14 @@ def store(report)
           szw.header_encryption = true
           # szw.multi_threading = false
           Dir.glob("#{destination_path}/*.csv").each do |f|
-            szw.add_data(File.open(File.join(local_destination_path, File.basename(f))).read, File.basename(f))
+            szw.add_data(::File.open(::File.join(local_destination_path, ::File.basename(f))).read, ::File.basename(f))
           end
         end
       end
       # for some reason we need a bit of sand after talking to zipcloak over PTY
```
Comment (awisema) [inline app/models/grda_warehouse/recurring_hmis_export.rb:146]: This must be an old comment?

**Review by awisema — APPROVED** [review]

File: app/models/grda_warehouse/recurring_hmis_export.rb (hunk 1 above)
Comment (eanders) [reply app/models/grda_warehouse/recurring_hmis_export.rb:97]: Just an echo to the console that the password was passed in to expect and then `interact` returns the pty to the running process.

File: app/models/grda_warehouse/recurring_hmis_export.rb (hunk 2 above)
Comment (eanders) [reply app/models/grda_warehouse/recurring_hmis_export.rb:146]: yes

**Pull request merged**